ebill_paynet/models/paynet_service.py

```diff
@@ -53,6 +53,10 @@ class PaynetService(models.Model):
         readonly=True,
     )
     active = fields.Boolean(default=True)
+    operation_timeout = fields.Integer(
+        string="HTTP Timeout",
+        help="Timeout for each HTTP (GET, POST) request in seconds."
```

Comment:
Does it deserve a default value to not have a blocked Odoo worker indefinitely?

Reply:
Maybe but how much is safe ?

Reply:
Ok, I have set one